hw2/05-form.js

```diff
@@ -1 +1,55 @@
-// Add your code here
+// Listen for when the entire document has been loaded before running the script
+document.addEventListener('DOMContentLoaded', () => {
+    // Access the student form and modal element from the document
+    const registrationForm = document.getElementById('studentForm');
+    const infoModalElement = document.getElementById('formModal');
+    const infoModal = new bootstrap.Modal(infoModalElement);
+  
+    // Function to retrieve and return data from the form fields
+    const extractFormData = () => ({
+      studentName: document.getElementById('name').value,
+      studentEmail: document.getElementById('email').value,
+      registrationStatus: document.getElementById('registrationStatus').value,
+      hasTakenPL: document.getElementById('PL').checked,  // Programming Languages
+      hasTakenOS: document.getElementById('OS').checked,  // Operating Systems
+      hasTakenFS: document.getElementById('FS').checked,  // Full Stack Development
+      additionalNotes: document.getElementById('additionalNotes').value
+    });
```

Comment:
This is a nice way to return data from a function instead of adding a return argument.